hammer_cli_reference/inventory.yaml

```
@@ -0,0 +1,10 @@
+---
+localhost:
+  hosts:
+    developer-system:
+      ansible_python_interpreter: "/usr/bin/python3"
+foreman-katello:
+  hosts:
+    foreman-katello.example.com:
+      ansible_user: "root"
```

Update: So far, I've managed to establish a connection to the Forklift instance with this hack:

```suggestion
  hosts:
   local-vm:
     ansible_connection: "ssh"
     ansible_host: 127.0.0.1
     ansible_port: 2222
     ansible_user: "vagrant"
     ansible_ssh_private_key_file: "/path/to/forklift/.vagrant/machines/_instance_/virtualbox/private_key"
```
However, the process fails with a command that requires root. I keep investigating...